README.md

```diff
@@ -129,19 +130,32 @@ sign-offs to merge PRs (and for large or contentious issues we may
 wait for more). For typos or other small fixes to documentation we
 relax this to a single sign-off.
 
-### Related Projects
+### The cats ecosystem
 
-There are many projects that integrate with Cats:
+#### Generic libraries to support pure functional programming
 
- * [Circe](https://github.com/circe/circe): pure functional JSON library.
+ * [cats-effects](https://github.com/typelevel/cats-effect): provide a standard `IO` type for the cats ecosystem
+ * [cats-mtl](https://github.com/typelevel/cats-mtl): ??????
```

Comment:
`cats-effect` and `cats-mtl` deserve their own category imo, as they are sub-modules of Cats, just like `cats-kernel`, but with a different release cycle.